pkg/catalog/routes.go

```diff
@@ -514,9 +529,150 @@ func (mc *MeshCatalog) listMeshServices() []service.MeshService {
 	return services
 }
 
+func (mc *MeshCatalog) destinationServicesFromTrafficTarget(t *target.TrafficTarget) ([]service.MeshService, error) {
+	sa := service.K8sServiceAccount{
+		Name:      t.Spec.Destination.Name,
+		Namespace: t.Spec.Destination.Namespace,
+	}
+	destServices, err := mc.GetServicesForServiceAccount(sa)
+	if err != nil {
+		return nil, errors.Errorf("Err finding Services for Service Account %#v: %v", sa, err)
```

Comment:
```suggestion
		return nil, errors.Errorf("Error finding Services for Service Account %#v: %v", sa, err)
```

Reply:
s/Err/Error so that it can be easily grepped in the logs and is consistent with other error logs.